src/main/java/seedu/duke/exception/InvalidCapAttained.java

```diff
@@ -0,0 +1,4 @@
+package seedu.duke.exception;
+
+public class InvalidCapAttained extends Exception {
```

Comment:
Maybe rename to InvalidCapCalculatedException?

Reply:
Seconded

Reply:
thirded, motion granted